lms/startup.py

```diff
@@ -39,8 +39,8 @@ def run():
         enable_third_party_auth()
 
     # Initialize Segment analytics module by setting the write_key.
-    if settings.SEGMENT_KEY:
-        analytics.write_key = settings.SEGMENT_KEY
+    if settings.LMS_SEGMENT_KEY:
+        analytics.write_key = settings.LMS_SEGMENT_KEY
```

Comment:
@bderusha is it true that the `analytics` module is never initialized in the CMS? I only see it being initialized for the LMS.